src/cramFile/slice/decodeRecord.js

```diff
@@ -6,12 +6,12 @@ import Constants from '../constants'
  * given a Buffer, read a string up to the first null character
  * @private
  */
-function readNullTerminatedStringFromBuffer(buffer) {
-  const zeroOffset = buffer.indexOf(0)
-  if (zeroOffset === -1) {
-    return buffer.toString('utf8')
+function readString(buffer) {
```

Comment:
could you restore the 'NullTerminated' part of the function name? Since this is a compression library, null-terminated is far from the only way to store a string.